content/events/2020-portland-oregon/prospectus.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Sponsorship Prospectus
+---
+
+## Post-Event Social: $6,000
+
+_Two sponsors may split this sponsorship. In case of a split, each sponsor get all benefits below in full unless otherwise noted._
+
+- Exclusive branding at event
+- 6 Tickets
+  - 3 tickets per sponsor if two sponsors split
+- Table presence at the venue
+- Logo on site and conference materials
+- A thank you announcement in opening and closing presentations
+
+## Gold: $3,000 (Limit 3)
+
+- Your logo on banner at conference site, conference materials, and website
+- A mention in all conference-related announcements
+- A thank you announcement in opening and closing presentations
+- A 2 minute promotional spot during talk intermissions
+- 5 tickets (max 2 at table at any time)
+- Table presence at the venue
+
+## Silver: $1,000 (Limit 10)
+
+- Your logo on conference materials and website
+- A thank you in opening and closing presentations
+- 2 tickets (max 2 at a table at any time)
+- Shared table presence at the venue
+
+## Sponsorships (check one):
+
+____    Post-Event Social $6,000
+
+____    Gold Sponsor $3,000
+
+____    Silver Sponsor $1,000
+
+Total Sponsorship Amount:       $_____________________________
+
+Invoicing Details:
+Company Name:__________________________________________________
+
+Contact Person:_________________________________________________
+
+Position/Title:________________________________________________
+ 
+Email: _________________________________________________________
+
+Company Address:________________________________________________
+
+City____________________________ ST:____________________________
+
+Zip:____________________________ Country:_______________________
+
+Sponsorships are accepted on a first-come, first-served basis.
+Sponsor must be paid by credit card upon acceptance.
+
+Signature:_________________     Signature:_________________
+
+Date: _____________________     Date: _____________________ 
+
+Printed Name ______________     Printed Name ______________
+
+Sponsor ___________________     Organizer _________________
+
+Please email the completed form to sponsors@cnpdx.us
```

Comment:
Is there a @kubernetescommunitydays.org alias to use here rather than @cnpdx.us?

Reply:
We don't have an alias at the moment AFAIK

Reply:
I've created the alias (sponsors-portland@kubernetescommunitydays.org) for your use. It will forward to the organizers alias.

Reply:
@KTan1226 thank you! can you remind me what the organizers' alias is?

Reply:
Sure thing! organizers-portland@kubernetescommunitydays.org

Reply:
thanks so much @KTan1226